filtoor/api
# [API] Refactor code + rewrite training code in python

### Summary
This PR accomplishes the following: 
1. change all python code to adhere to pylint standards
2. refactor methods to be used by training code and server code into helpers.py
3. rewrite training code from .js to .py 
4. add more robustness to db item query by removing the item if a field we want does not exist (ensures robustness of structure of data to a certain extent) 

### Test Plan
1. start server
2. ensure curls work and a response is returned 
3. run python training script and ensure model is updated
